tests/release_notes/model/test_custom_chapters.py

```diff
@@ -149,17 +149,16 @@ def test_populate_none_duplicity_scope(custom_chapters, mocker):
 # from_json
 
 
-def test_custom_chapters_from_json():
+def test_custom_chapters_from_yaml_array():
     custom_chapters = CustomChapters()
-    json_string = """
-    [
+    yaml_array_in_string = [
         {"title": "Breaking Changes 💥", "label": "breaking-change"},
         {"title": "New Features 🎉", "label": "enhancement"},
         {"title": "New Features 🎉", "label": "feature"},
         {"title": "Bugfixes 🛠", "label": "bug"}
     ]
```

Comment:
This si correct yaml_array_in_string. You provided a Python list.

This is how would look like a yaml in a string. Please unite the terminology.
yaml_array_in_string = """
- title: "Breaking Changes 💥"
  label: "breaking-change"
- title: "New Features 🎉"
  label: "enhancement"
- title: "New Features 🎉"
  label: "feature"
- title: "Bugfixes 🛠"
  label: "bug"
"""

Reply:
sure, but from workflow it has to be transferred by string.
I saw it in documentation.